### ISSUE TITLE
How do I set the http response code?

### ISSUE THREAD

**Issue by danilvoe** [72pulses/apidoco#86]
It is necessary to reflect the http code in the examples of requests, how can this be done?

**Comment by alameenkhader** [comment]
You are right. We need to have the http code the example requests and responses. We do not have that feature as of now. I added your request to the pipeline. Thanks
